### PR TITLE
Local-DNS-resolver reworked

### PR DESCRIPTION
**Pull request by ghost**
Merged everything in one installer.
Updated `unbound.conf` configuration.

### CODE REVIEW

**Comment by angristan** [comment]
Hey, thanks for the PR, it's indeed better to do it like this. 

I'll take a look later, can you tell me what's `unbound.conf.new`?

**Comment by cezar97** [comment]
Well, I left it in the repo because of the comments for every config the installer writes in the `unbound.conf` file. I thought that there were to much lines of comments to put in the `unbound-installer.sh` but integrating them shuold be better anyway.

File: unbound-install.sh [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+				echo "$VERSION_ID not supported"
+				while [[ $CONTINUE != "y" && $CONTINUE != "n" ]]; do
+					read -p "Continue anyway? [y/n]: " -e CONTINUE
+				done
+		if [[ "$CONTINUE" = "n" ]]; then
+			echo "Ok, bye !"
+			exit 4
+		fi
+	fi	
+elif [[ -e /etc/centos-release || -e /etc/redhat-release || -e /etc/system-release && ! -e /etc/fedora-release ]]; then
```
Comment (angristan) [inline unbound-install.sh]: `/etc/fedora-release` is not needed here, is it?

Comment (cezar97) [reply unbound-install.sh]: Fedora uses dnf but you're right: it's not needed as yum redirects to dnf for what I know.

Comment (angristan) [reply unbound-install.sh]: I was pointing this out because of line 28 😄

Comment (cezar97) [reply unbound-install.sh]: I'm still learning bash, so for OS detection I used your functions in the openvpn-install.sh.

Comment (angristan) [reply unbound-install.sh]: Sorry I didn't see the `!`.... Nevermind

**Comment by angristan** [comment]
Is there a reason why you removed arch-bind.sh?

**Comment by cezar97** [comment]
Mainly because bind is primarly most used as an authoritative dns server and unbound as a recursive one, also seems that unbound is more lightweight, faster and most secure.
Personally I would choose powerdns as authoritative and unbound as recurisve btw, but I can re-add arch-bind.sh if you like so.

**Comment by angristan** [comment]
Let's go full unbound then 👍 

Thanks haha

File: unbound-install.sh [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+clear
```
Comment (angristan) [inline unbound-install.sh]: The script is pretty straightforward here (no complex menus, etc), so I don't think it's needed here

**Comment by angristan** [comment]
Okay so I kinda rewrote the script.

Indeed, default options of Unbound are fine and DNSSEC is even enabled by default, except on Arch. 

Also, options like `qname-minimization` or `so-reuseport` aren't available on older versions of Unbound, so it just fails to start if it's not recognized.

I separated more of the install/configuration commands, every OS has its own way to setup Unbound (like the folders) so we can't mutualise everything.

But thanks for the PR, at least we have everything in one script :)

I ran the the script on all OS and I can confirm this works now.

**Pull request merged**